src/_SignAssembly.targets

```
@@ -0,0 +1,40 @@
+<Project>
+  <!--
+      Conditional strong name signing of netstandard assemblies for better cross-runtime compatibility.
+      
+        Ref: https://docs.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/6.0/strong-name-signing-exceptions#recommended-action
+     -->
+
+  <!--
+    NOTE: GraphQLParser.Tests depends on netstandard 2.1 for netcore3.1 and net5.0 tests, requiring a signed friend assembly for internals visible to.
+  -->
+  <PropertyGroup Condition="'$(TargetFramework)'=='net6'">
+    <SignAssembly>false</SignAssembly>
+  </PropertyGroup>
+
+  <PropertyGroup  Condition="'$(TargetFramework)'!='net6'">
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>$(SolutionDir)graphql.snk</AssemblyOriginatorKeyFile>
```

```suggestion
    <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)graphql.snk</AssemblyOriginatorKeyFile>
```
Maybe this will help the codeql to build properly